r4/resource-apis/uscore501-patient/service.bal

```diff
@@ -25,81 +25,83 @@ import ballerinax/health.fhir.r4.uscore501;
 import ballerina/log;
 import ballerina/http;
 
-# Generic type to wrap all implemented profiles. 
+# Generic type to wrap all implemented profiles.
 # Add required profile types here.
 # public type Patient r4:Patient|r4:USCorePatient;
 public type Patient r4:Patient | uscore501:USCorePatientProfile;
 
 //add implemented profiles to this map. profileURL:implementation
 isolated final map<PatientSourceConnect> profileImpl = {
-    "http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient": new Uscore501PatientSourceConnect()
+    "http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient": new UscorePatientSourceConnect()
```

Comment:
Let's keep it as Uscore501PatientSourceConnect. `uscore501` is the term being used across.

Reply:
removed in 
3484721